esp/scm/ws_topology.ecm

```diff
@@ -316,6 +316,15 @@ ESPStruct TpClusterNameType
     bool   IsDefault;
 };
 
+ESPStruct TpStoragePlan
+{
+    string Name;
+    string Hosts;
+    string Prefix;
+    string Mount;
```

Comment:
Not sure where this ('Mount') came from?
there the planes are mounted is really an internal matter.

Reply:
I saw it when I dry-run:
planes:
        - name: hpcc-data-plane
          mount: "/var/lib/HPCCSystems/hpcc-data"
        - name: hpcc-spill-plane
          mount: "/var/lib/HPCCSystems/hpcc-spill"

I can remove it since it is for internal.

Reply:
right, yes it's generated, but used internally, don't think we'd want to return it to a client.